src/amber/server/server.cr

```diff
@@ -74,7 +74,7 @@ module Amber
           server.listen(settings.port_reuse)
           exit
         rescue e : Errno
-          if e.message == "accept: Too many open files"
+          if e.errno == Errno::E2BIG
```

Comment:
Is E2BIG equivalent to "accept: Too many open files"?

Reply:
Yes, I looked it up in Crystal sources.